webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardList.vue

```diff
@@ -134,11 +134,8 @@ export default {
         this.searchPeople();
         return;
       }
-      this.startTypingKeywordTimeout = Date.now();
-      if (!this.loadingPeople) {
-        this.loadingPeople = true;
-        this.waitForEndTyping();
-      }
+      this.searchPeople();
```

Comment:
startSearchAfterInMilliseconds: 600,
    endTypingKeywordTimeout: 50,
    startTypingKeywordTimeout: 0,

should be removed